samples/boards/tlsr9x/pwm_pool/boards/tlsr9528a.overlay

```diff
@@ -18,8 +18,8 @@
 &pwm0 {
 	/* On board RGB LEDs */
 	pinctrl-ch0 = <&pwm_ch0_pe7_default>;
-	pinctrl-ch2 = <&pwm_ch1_pd1_default>;
-	pinctrl-ch1 = <&pwm_ch2_pd0_default>;
+	pinctrl-ch1 = <&pwm_ch1_pd1_default>;
```

Comment:
Since corresponding .dts is already in board .dts do we still need these samples overlays? For me it seems confusing. If we keep board overlays we have possibility to quick check Matter functionality.

Reply:
It is duplicate info. So we can delete overlays.